example/deploy/csi-mock-driver-deployment.yaml

```diff
@@ -0,0 +1,125 @@
+# This YAML file contains the deployment of the CSIDriver
+# including node-driver-registrar, external-provisioner, external-snapshotter
+# external-resizer and mock-driver
+#
+# This YAML file also includes deployment of the CSIDriver, StorageClass
+# VolumeSnapshotClass for the mock driver.
+
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: csi-mockplugin
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: csi-mockplugin
+  template:
+    metadata:
+      labels:
+        app: csi-mockplugin
+    spec:
+      serviceAccount: csi-mock
+      containers:
+        - name: csi-provisioner
+          image: gcr.io/k8s-staging-csi/csi-provisioner:canary
+          args:
+            - "--csi-address=$(ADDRESS)"
+            - "--leader-election"
```

Comment:
We can probably disable leader election for all the sidecars since the mock driver is not resilient to restarts anyway.

Reply:
Done